api/topology/v1/topology_api.proto

```diff
@@ -158,3 +159,29 @@ message Edge {
   // edge.
   map<string, google.protobuf.Value> metadata = 4;
 }
+
+message TopologyObject {
```

Comment:
Discussing IRL but TopologyObject makes me think this is an instance of Topology. This is pretty minor and I could just be alone here since I can't really think of something better at the moment. Any thoughts on this @danielhochman ?